tests/components/xiaomi_ble/test_event.py

```diff
@@ -198,6 +249,7 @@ async def test_events(
         state = hass.states.get(meas["entity"])
         attributes = state.attributes
         assert attributes[ATTR_FRIENDLY_NAME] == meas[ATTR_FRIENDLY_NAME]
-        assert attributes[ATTR_EVENT_TYPE] == meas[ATTR_EVENT_TYPE]
+        if ATTR_EVENT_TYPE in attributes:
```

Comment:
We try to avoid branching in tests. Usually it means splitting the test or adjusting the params to avoid the need to branch